static/src/javascripts-legacy/projects/commercial/modules/creatives/add-tracking-pixel.js

```diff
@@ -1,14 +1,9 @@
-define([
-    'common/utils/template',
-    'raw-loader!commercial/views/creatives/tracking-pixel.html'
-], function (
-    template,
-    trackingPixelStr
-) {
-    var trackingPixelTpl = template(trackingPixelStr);
-    function addTrackingPixel($adSlot, url) {
-        $adSlot.before(trackingPixelTpl({ url: encodeURI(url) }));
-    }
+define(function () {
 
     return addTrackingPixel;
+
+    function addTrackingPixel(url) {
```

Comment:
It is not added to the DOM, because that is not necessary. The image will be loaded as soon as the `src` is assigned 😄